src/main/resources/templates/home.html

```diff
@@ -1,15 +1,37 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml"
       xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      xmlns:th="https://www.thymeleaf.org"
       layout:decorate="layout">
     <head>
         <title>Home</title>
+        <link th:href="@{/css/application.css}" rel="stylesheet" />
+        <link th:href="@{/css/home-page-style.css}" rel="stylesheet" />
+
     </head>
     <body>
         <section layout:fragment="content">
-            <h1>Welcome to Whale Spotting!</h1>
```

Comment:
Pages should have an `h1` title somewhere, maybe this could go above the `<section>`?